language/tools/move-cli/tests/testsuite/events_emit_view/args.exp

```diff
@@ -5,22 +5,22 @@ Publishing a new module 00000000000000000000000000000002::Events
 Command `run src/scripts/emit.move --signers 0xA --args 5 -v`:
 Compiling transaction script...
 Emitted 1 events:
-Emitted Value(Container(Struct(RefCell { value: [U64(5)] }))) as the 0th event to stream [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10]
+Emitted [5, 0, 0, 0, 0, 0, 0, 0] as the 0th event to stream [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 10]
```

Comment:
This is a rather unfortunate side effect of this change: values are already serialized when they leave the VM. Let me see if I can do something better here.

Reply:
I think this is ok (unless there's an easy). This printing is already very raw, I think there is a TODO to make it human-readable.